recipes/voms-clients/bld.bat

```diff
@@ -0,0 +1,20 @@
+call mvn -Dmaven.javadoc.skip=true -Dvoms-clients.libs="%PREFIX%\share\voms-clients\lib" package
+if errorlevel 1 exit 1
+
+dir target
+7za x -so "target\voms-clients.tar.gz" | 7za x -si -aoa -ttar > NUL 2>&1
+if errorlevel 1 exit 1
+
+dir
+dir voms-clients
+dir voms-clients\bin
```

Comment:
I believe these are only for debugging, right?

Reply:
Yes but I would rather leave them in as I haven't managed to get the Windows build working yet (I skip for now).